### DISCUSSION TITLE
Error compilation with platform.local.txt

### DISCUSSION THREAD

**Discussion by DaTTcz** [Bodmer/TFT_eSPI#2203] [General]
When I copy the platform.local.txt file I get an error when compiling with other libraries. e.g. ESPAsyncWebServer. md5.h error
What am I doing wrong ? Thank you and sorry for my English.

**Comment by Bodmer** [comment]
This approach to a user setup in the sketch does seem to cause issues with some support libraries and when it does work it is only with the ESP32.

The only fix is to use the original setup file approach inside the library.
